sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientBuilderExtensions.cs

```diff
@@ -39,8 +39,8 @@ public static IAzureClientBuilder<ConfigurationClient, ConfigurationClientOption
         /// <summary> Registers a <see cref="ConfigurationClient"/> instance. </summary>
         /// <param name="builder"> The builder to register with. </param>
         /// <param name="configuration"> The configuration values. </param>
-        [RequiresUnreferencedCode("Requires unreferenced code until we opt into EnableConfigurationBindingGenerator.")]
```

Comment:
Could you actually revert the changes to this file and then re-run the export api script for appconfiguration? It turns out the messages are part of the API contract and can no longer be changed.